ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -317,3 +319,38 @@ test-suite test-infra
                        -Wmissing-export-lists
                        -Wno-unticked-promoted-constructors
                        -fno-ignore-asserts
+
+benchmark bench-storage
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      bench-storage
+  main-is:             Main.hs
+  other-modules:
+                       Bench.Ouroboros.Consensus.Storage.LedgerDB.HD
+
+  build-depends:       base
+                     , bytestring
+                     , containers
+                     , deepseq
+                     , mtl
+                     , QuickCheck
+                     , QuickCheck-GenT
+                     , strict-containers
+                     , tasty-bench
+                     , tasty-quickcheck
+
+                     , ouroboros-consensus
+                     , ouroboros-consensus-test
+
+  default-language:    Haskell2010
+  ghc-options:         -Wall
+                       -Wcompat
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+                       -Wpartial-fields
+                       -Widentities
+                       -Wredundant-constraints
+                       -Wmissing-export-lists
+                       -Wno-unticked-promoted-constructors
+                       -fno-ignore-asserts
+                       -rtsopts
+                       "-with-rtsopts=-A32m"
```

Comment:
Why did we set the allocation area to this? My guess: to prevent GC from interfering with the benchmarks. If so, did we confirm that this setting works? Maybe we should add a comment on this.